service_state.go

```diff
@@ -0,0 +1,123 @@
+package baps3
+
+// The part of the BAPS3 connector code responsible for updating the
+// internal state.
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// ServiceState is the struct of all known state for a BAPS3 service.
+// TODO(CaptainHayashi): possibly segregate more by feature, so elements not
```

Comment:
I'd say leave the `TODO` in.  It isn't harming anyone, and it is something that could be fixed, even if we're not fixing it _in this prq_